indico/modules/events/registration/models/form_fields.py

```diff
@@ -59,13 +58,45 @@ def __repr__(self):
         return '<RegistrationFormFieldData({}, {})>'.format(self.id, self.field_id)
 
 
+class RegistrationFormFieldSemanticNames(db.Model):
```

Comment:
Just saw that you updated this PR. I don't think adding a new table for this is a good idea - instead, I'd rather put a String column directly on `RegistrationFormItem`.

Reply:
Well, the idea is to add a autocompleter for know "semantic names" as these should be reused across conferences and registration forms.  But I think we should postpone this for after the regform angular cleanup anyway.

Reply:
I don't like the idea of autocompleting this across events. And within an event having the extra table is overkill - getting it straight from the form_items table doesn't add noticable overhead

Reply:
But yes, better to keep this for the future until after we got rid of the Angular mess :)